lib/game_flow.ex

```diff
@@ -8,46 +8,32 @@ defmodule GameFlow do
   end
   
   def loop(game) do
-    game |> Console.print_board
-    game |> turn_message
-    %current_player{} = game.current_player
-    case current_player do
-      Player ->
-        game |> get_user_move()
-      Computer ->
-        game |> Computer.move()
-    end
+    game 
+    |> Console.print_board
+    game 
+    |> Message.turn 
+    |> Console.print()
+    game.current_player 
+    |> Player.Move.move(game) 
+    |> Game.mark_spot(game)
+    |> Game.change_turn
```

Comment:
This is nice!

Did you think about dependency injection here?  You could have a testable loop if you hand in `Message`, `Console`, `Game`, or `Player`.  I'm not sure if it's _necessary_, but it could be _possible_